src/plugins/discover/public/application/main/components/sidebar/discover_sidebar.tsx

```diff
@@ -104,6 +104,7 @@ export function DiscoverSidebarComponent({
   closeFlyout,
   editField,
   viewMode,
+  availableFields$,
```

Comment:
Seems like it is redundant leftover.

Reply:
Thanks for catching that! Removed here https://github.com/elastic/kibana/pull/123003/commits/1abf9e38339486698dc85c93de2622d7312a5bef